app/views/catalog/_home.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class='jumbotron'>
+  <div class='row'>
+    <h1><%= t('arclight.views.home.heading') %></h1>
```

Comment:
What would you think about adding `text-center` class here to center this heading?